praw/models/preferences.py

```diff
@@ -0,0 +1,185 @@
+"""Provide the Preferences class."""
+from json import dumps
+
+from ..const import API_PATH
+
+
+class Preferences(object):
+    """A class for Reddit preferences.
+
+    The Preferences class provides access to the Reddit preferences of the
+    currently logged in user.
+    """
+
+    def __init__(self, reddit):
+        """Create a Preferences instance.
+
+        :param reddit: The Reddit instance.
+        """
+        self._reddit = reddit
+
+    def __call__(self):
```

Comment:
Huh, I thought one of the linting tools would have caught this. I guess I just overlooked the rule about alphabetical ordering because of this assumption.

Do you know if one of the tools that PRAW uses supports this? I might look into it.